addons/packages/fluxcd-source-controller/0.24.4/README.md

```diff
@@ -0,0 +1,194 @@
+# Fluxcd-source-controller
+
+The source-controller is a Kubernetes operator, specialised in artifacts acquisition
+from external sources such as Git, Helm repositories and S3 buckets.
+The source-controller implements the
+[source.toolkit.fluxcd.io](https://github.com/fluxcd/source-controller/tree/master/docs/spec/v1beta1) API
+and is a core component of the [GitOps toolkit](https://toolkit.fluxcd.io).
+
+## Configuration
+
+The fluxcd-source-controller package has following configurable properties.
+
+| Value           | Required/Optional |                           Description                                                |
+|-----------------|-------------------|--------------------------------------------------------------------------------------|
+| `namespace`     | Optional          | Sets namespace for k8s objects where resources of source-controller will be created. |
+| `limits_cpu`    | Optional          | Sets maximum usage of cpu for source-controller deployment.                          |
+| `limits_memory` | Optional          | Sets maximum usage of memory for source-controller deployment.                       |
+| `no_proxy`      | Optional          | Set domains for which no proxying should be performed                                |
+| `https_proxy`   | Optional          | Set secure proxy connection information                                              |
+| `http_proxy`    | Optional          | Set unsecure proxy connection information                                            |
+
+```yaml
+namespace: source-system
```

Comment:
The `0.24.4` version of the package is installing into the `source-system` namespace. This is an intentional change? If so, the README and the e2e tests will need to be updated to reflect this as they still reference `flux-system`.

Reply:
Done